client/ayon_core/hosts/maya/plugins/publish/validate_maya_units.py

```diff
@@ -132,5 +132,5 @@ def repair(cls, context):
 
         cls.log.info("Setting time unit to match project")
         # TODO replace query with using 'context.data["folderEntity"]'
-        folder_entity = get_current_project_folder()
+        folder_entity = get_current_folder_entity()
```

Comment:
This would be better replaced with `context.data["folderEntity"]` as the `# TODO` describes above it.